cmd/info.go

```diff
@@ -59,6 +59,11 @@ $ juicefs info -i 100`,
 				Aliases: []string{"r"},
 				Usage:   "get summary of directories recursively (NOTE: it may take a long time for huge trees)",
 			},
+			&cli.BoolFlag{
+				Name:    "fast",
```

Comment:
Should we use the faster one as default? then we could add a option called `--accurate` or `--strict`

Reply:
Reasonable

Reply:
Done